config.go

```diff
@@ -18,6 +18,17 @@ type TestcontainersConfig struct {
 // of the TestcontainersConfig struct
 func ReadConfig() TestcontainersConfig {
 	cfg := config.Read()
+
+	if cfg.RyukDisabled {
```

Comment:
Will this be printed anytime that a user reads the config? As this method is public API, it would be a bit noisy seeing that message that much.

Reply:
I, personally, don't see a problem, as this is a kind of warning message. What do you think could be a better approach?